<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QHofflinePath</class>
 <widget class="QDialog" name="QHofflinePath">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QHofflinePath</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Offline Setting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Path</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="offlinepathlineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="offlineopenpushButton">
          <property name="text">
           <string>OPEN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="readsamepicturecheckBox">
          <property name="text">
           <string>load same picture</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="savepushButton">
          <property name="text">
           <string>save</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
